.github/workflows/test.yml

```diff
@@ -363,8 +365,9 @@ jobs:
       - name: rust cache
         uses: Swatinem/rust-cache@v2
         with:
-          prefix-key: "v1-${{ runner.os }}-rust-cache"
-          shared-key: "cli-e2e"
+          prefix-key: "v3"
+          shared-key: "rust-cache"
+          cache-all-crates: true
```

Comment:
don't think this will do much based on their docs :/ (https://github.com/Swatinem/rust-cache). What it adds as caching of any CI-specific rust crates, of which I think we have none?